e2e/visual-regression/all.e2e.ts

```diff
@@ -0,0 +1,23 @@
+const storyName = 'All Combinations';
```

Comment:
We also discussed testing all other states, because on some components it does not make sense to have `All Combinations`. Also, this would not include compositions.

Do you plan to still add to this PR, or it would be another one?

Reply:
I didn't plan for that, but it doesn't seem hard to introduce here.

But because the name isn't static, we either:
1. Screenshot all stories
2. Have an array of story names here
3. Write separate, specific spec/test file dedicated to each component

I feel like 3 is probably better, in which case it can be done after this PR. wdyt?

Reply:
Sure, no problem, then title of this PR should probably be like: `test: add visual regression support for all combinations`.

Reply:
Well support is also there for other spec/tests, I just happened to check the support works for "All Combinations" stories.

Perhaps `test: add visual regression support and spec all combinations`?

Reply:
Sure, thanks works 👍